Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -6753,8 +6753,11 @@ def __init__(self, outputs_prefix=None,
         # type: (str, object, object, list, str, object, IndicatorsTimeline, Common.Indicator, bool, bool, bool, ScheduledCommand, list, int, str, List[Any]) -> None  # noqa: E501
         if raw_response is None:
             raw_response = outputs
-        if outputs is not None and not isinstance(outputs, dict) and not outputs_prefix:
-            raise ValueError('outputs_prefix is missing')
+        if outputs is not None:
+            if not isinstance(outputs, dict) and not outputs_prefix:
+                raise ValueError('outputs_prefix is missing')
+            if outputs_prefix == '.':
```

Comment:
@bziser I didn't check if the output isn't a dict because even if the output is a dict we won't accept the period prefix